src/android/CDVMFPPush.java

```diff
@@ -266,9 +279,30 @@ public void onResume(boolean multitasking) {
     public void onPause(boolean multitasking) {
         super.onPause(multitasking);
         pushLogger.debug("In onPause");
+
+        if (!ignoreIncomingNotifications) return;
+
         if(MFPPush.getInstance() != null) {
             MFPPush.getInstance().hold();
         }
     }
 
-}
+    /**
+     * Change the plugin's default behavior to ignore handling push notifications
+     * @param ignore boolean parameter for the 'ignore notifications' behavior
+     */
+    public void setIgnoreIncomingNotifications(boolean ignore) {
+        pushLogger.debug("In setIgnoreIncomingNotifications : ignore = " + ignore);
+        ignoreIncomingNotifications = ignore;
+
```

Comment:
@dgonz7 @AntonAleksandrov : Should we consider the case where the user called registerNotificationsCallback and then later on wants to call this method to have the plugin ignore incoming push notifications (maybe temporarily)?